Check if the path passed is a directory.

Change 3 of 6 from
https://sourceforge.net/p/gelide/code/ci/b8da37d825b9548ba6548f96ce8efb84cf068001/
--- src/core/dat_reader_factory.cpp
+++ src/core/dat_reader_factory.cpp
@@ -30,8 +30,8 @@ CDatReader* CDatReaderFactory::getDatReader(const Glib::ustring& p_file){
 	int l_fsize;
 	CDatReader* l_reader = NULL;
 
-	// El fichero debe ser válido
-	if(!p_file.size())
+	// El fichero debe ser válido y no puede ser un directorio
+	if(!p_file.size() || Glib::file_test(p_file, Glib::FILE_TEST_IS_DIR))
 		return NULL;
 
 	l_file.open(p_file.data());
